flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-values/src/main/java/com/ververica/cdc/connectors/values/sink/ValuesDataSinkHelper.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2023 Ververica Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ververica.cdc.connectors.values.sink;
+
+import com.ververica.cdc.common.event.DataChangeEvent;
+import com.ververica.cdc.common.event.Event;
+import com.ververica.cdc.common.event.SchemaChangeEvent;
+import com.ververica.cdc.runtime.state.TableSchemaState;
+
+/** A helper class for {@link ValuesDataSink} to process {@link Event}. */
+public class ValuesDataSinkHelper {
+
+    /** convert Event to String for {@link ValuesDataSink} to display detail message. */
+    public static String convertEventToStr(Event event, TableSchemaState tableSchemaState) {
+        if (event instanceof SchemaChangeEvent) {
+            return event.toString();
+        } else if (event instanceof DataChangeEvent) {
+            DataChangeEvent dataChangeEvent = (DataChangeEvent) event;
+            String eventStr =
+                    "DataChangeEvent{"
+                            + "tableId="
+                            + dataChangeEvent.tableId()
+                            + ", before="
+                            + tableSchemaState.getFields(dataChangeEvent.before())
+                            + ", after="
+                            + tableSchemaState.getFields(dataChangeEvent.after())
+                            + ", op="
```

Comment:
Could you add a tests for this change ?

Reply:
Address it and add in ValuesDataSinkHelperTest.